### DISCUSSION TITLE
Smart reply suggestion in the bottom of chat

### DISCUSSION THREAD

**Discussion by M-ZubairAhmed** [mattermost/community#73] [Fun Ideas]
The concept is similar to AI generated reply based on the incoming messages, may be list of 2 - 3 suggested replies to quickly select and send. 

Incoming : Hey can we get on a zoom call
Suggested : `OK sure` `Let me get back to you`

**Comment by chetanyakan** [comment]
This should be configurable for each user, whether they want to see such messages.

Self-hosted Mattermost instances should have an option to apply their own recommendation system.

**Reply by M-ZubairAhmed** [reply]
Yes definitely more secure that way 🔒

**Comment by esethna** [comment]
Love this idea! Or take a beat from Gmail and help with autocompleting sentences

**Reply by M-ZubairAhmed** [reply]
haha yeah it would be cool too 💯
